lua/autorun/client/cl_focus.lua

```diff
@@ -0,0 +1,26 @@
+local tabbedOutList = {}
+local Icon = Material( "icon16/application_double.png" ) -- the afk icon
+
+timer.Create( "tabNetTimmer", 1, 0, function()

```

Comment:
I'd lower this to like ~5 seconds considering a server with 20+ people would mean 20 net messages per second.
What you could also do is make it only send a net message when the status is switched instead of constantly letting the server knows, I'd highly recommend doing so.